tests/net/test_server.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+from circuits.web.servers import BaseServer
+from circuits.net.sockets import BUFSIZE
+
+class MockClass(BaseServer):
+    def __init__(self, *args, **kwargs):
```

Comment:
Please replace `__init__` with a simple `pass`, as the syntax is not python 2 compatible.

Reply:
replaced `__init__` with a `pass`